docs/covsearch.rst

```diff
@@ -82,6 +82,14 @@ Optional
 | ``algorithm``                               | The search :ref:`algorithm<algorithm_covsearch>` to use               |
 |                                             | (default is `'scm-forward-then-backward'`)                            |
 +---------------------------------------------+-----------------------------------------------------------------------+
+| ``max_eval``                                | Limit the number of function evaluations to 3.1 times that of the     |
+|                                             | base model. Default is False.                                         |
++---------------------------------------------+-----------------------------------------------------------------------+
+| ``adaptive_scope_reduction``                | Stash all non-significant parameter-covariate effects to be tested    |
```

Comment:
I think this sentence is slightly confusing. Perhaps: "Stash all non-significant parameter-covariate effects at each step in the forward search for later evaluation. As soon as all significant effects have been tested the stashed effects gets evaluated in a normal forward approach."